test/common/http/conn_manager_impl_test.cc

```diff
@@ -833,6 +833,7 @@ TEST_F(HttpConnectionManagerImplTest, MultipleFilters) {
       .WillOnce(InvokeWithoutArgs([&]() -> Http::FilterHeadersStatus {
         EXPECT_EQ(route_config_provider_.route_config_->route_,
                   decoder_filter1->callbacks_->route());
+        EXPECT_EQ(ssl_connection_.get(), decoder_filter1->callbacks_->ssl());
```

Comment:
The way this test is setup above, ssl_connection_.get() is going to be nullptr. Can we make this an "SSL" connection and then this test would be more useful?